apitools/org.eclipse.pde.api.tools/META-INF/MANIFEST.MF

```diff
@@ -2,12 +2,12 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %pluginName
 Bundle-SymbolicName: org.eclipse.pde.api.tools;singleton:=true
-Bundle-Version: 1.3.800.qualifier
+Bundle-Version: 1.3.900.qualifier
 Bundle-Vendor: %providerName
 Bundle-Localization: plugin
 Require-Bundle: org.eclipse.core.runtime;bundle-version="[3.29.0,4.0.0)",
  org.eclipse.pde.core;bundle-version="[3.17.200,4.0.0)",
- org.eclipse.jdt.core;bundle-version="[3.36.0,4.0.0)",
+ org.eclipse.jdt.core;bundle-version="[3.42.100,4.0.0)",
```

Comment:
As https://github.com/eclipse-jdt/eclipse.jdt.core/pull/3900 increments the version of jdt.core this should be updated to that new version.